x-pack/solutions/security/plugins/security_solution/public/management/components/console/types.ts

```diff
@@ -67,6 +67,11 @@ export interface CommandArgDefinition {
    * the argument's value.
    */
   SelectorComponent?: CommandArgumentValueSelectorComponent;
+
+  /**
+   * If defined, the selector will use `true` as the default value which results in empty array as args value
+   */
+  selectorStringDefaultValue?: boolean;
```

Comment:
I got a bit confused by this new prop's name. Could you describe what its purpose is?

Is it used only when processing commands from Paste action or selection from input history? and only during `SelectorComponent` initialization? or checked everytime the user input changes (ex. on typing).